app/exports/persona_export.yml

```diff
@@ -0,0 +1,88 @@
+common_columns:
+  - candidate_id
+  - support_reference
+  - program_type
+  - rejection_reason
+  - structured_rejection_reasons
+  - application_state
+  - course_code
+  - provider_code
+  - nationality
+  - provider_type
+
+custom_columns:
+  age:
+    type: integer
+    description: The candidate’s age
+
+  candidate_postcode:
+    type: string
+    description: The candidate’s postcode
+    example: SW1P 3BT
+
+  provider_postcode:
+    type: string
+    description: The candidate’s postcode
+    example: SW1P 3BT
+
+  site_postcode:
+    type: string
+    description: The site’s postcode
+    example: SW1P 3BT
+
+  site_region:
+    type: string
+    description: The site’s region
+    enum:
+      - east_midlands
+      - eastern
+      - london
+      - no_region
+      - north_east
+      - north_west
+      - scotland
+      - south_east
+      - south_west
+      - wales
+      - west_midlands
+      - yorkshire_and_the_humber
+
+  accrediting_provider_type:
+    type: string
+    description: The accrediting provider type
+    enum:
+      - scitt
+      - university
+      - hei
```

Comment:
And these please

Reply:
This should always be university or scitt.